### PR TITLE
refactor(csv): format test names

### PR DESCRIPTION
**Pull request by timreichen**
Ref: https://github.com/denoland/deno_std/issues/3754

### CODE REVIEW

File: csv/csv_parse_stream_test.ts [hunk 1]
```diff
@@ -57,97 +57,93 @@ Deno.test({
...
-        separator: ";",
-      },
-      {
-        name: "MultiLine",
-        input: `"two
+// These test cases were originally ported from Go:
+// https://github.com/golang/go/blob/go1.12.5/src/encoding/csv/
+// Copyright 2011 The Go Authors. All rights reserved. BSD license.
+// https://github.com/golang/go/blob/master/LICENSE
+const testCases = [
```
Comment (iuioiua) [inline csv/csv_parse_stream_test.ts:64]: I don't mind this change but I think it should be in a separate PR. It's best to keep PRs as lean and focused as possible.

**Review by iuioiua — CHANGES_REQUESTED** [review]
Mostly LGTM once my comment and use of "checks" is addressed.

File: csv/csv_parse_stream_test.ts (hunk 1 above)
Comment (timreichen) [reply csv/csv_parse_stream_test.ts:64]: Ok, I'll revert the change.

**Comment by timreichen** [comment]
> Mostly LGTM once my comment and use of "checks" is addressed.

I updated the test names

**Review by iuioiua — APPROVED** [review]
LGTM! Thank you.

Only thing is that @kt3k stated previously that he'd prefer class instances in test names to omit their brackets. This PR is fine to me as it is now. Having the brackets to me, while camel cased, tells me that it's a class instance.

**Review by kt3k — APPROVED** [review]
LGTM
